### PR TITLE
kodi-cli: Init at 1.1.1

### PR DESCRIPTION
**Pull request by pstn**
###### Motivation for this change
Added the handy utility `kodi-cli` for controlling remote kodi instances with a cli.
I was on the fence myself about PRing this, so I asked in #nixos. A few people wanted to have it, so here it is.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by worldofpeace — CHANGES_REQUESTED** [review]
A lot of the indentation and whitespace isn't up to par with what's in nixpkgs. 
See: https://nixos.org/nixpkgs/manual/#sec-syntax

File: pkgs/applications/video/kodi-cli/default.nix
Comment (worldofpeace) [inline pkgs/applications/video/kodi-cli/default.nix]: Something more popular in nixpkgs would be
```suggestion
  name = "kodi-cli-${version}";
```

And move the `version = "1.1.1";` into the set doing away with the `let in`.

File: pkgs/applications/video/kodi-cli/default.nix
Comment (worldofpeace) [inline pkgs/applications/video/kodi-cli/default.nix]: You'll need to add yourself to the https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix

File: pkgs/applications/video/kodi-cli/default.nix
Comment (symphorien) [reply pkgs/applications/video/kodi-cli/default.nix]: Since https://github.com/NixOS/nixpkgs/pull/49398 it is enough to write
```
pname = "kodi-cli";
version = "1.1.1";
```

Comment (worldofpeace) [reply pkgs/applications/video/kodi-cli/default.nix]: @symphorien Oh, I wasn't aware that hit master yet.

**Comment by pstn** [comment]
Whoopsie. Made a little git mess. Please disregard until I clean it up.

File: pkgs/applications/video/kodi-cli/default.nix
Comment (pstn) [reply pkgs/applications/video/kodi-cli/default.nix]: It's merged but only in the `name` version, not in the `pname` version.
When I use `name`, my package builds fine, but with `pname` it breaks.
I suggest to just leave this unmerged, until the other change is merged and not add a package that will break again in a few days.

File: pkgs/applications/video/kodi-cli/default.nix
Comment (pstn) [reply pkgs/applications/video/kodi-cli/default.nix]: I'm already in there somewhere. Wouldn't @GrahamcOfBorg complain otherwise btw?

File: pkgs/applications/video/kodi-cli/default.nix
Comment (worldofpeace) [inline pkgs/applications/video/kodi-cli/default.nix]: The changes @symphorien was speaking of for the derivation names isn't in master yet so this should be

`name = "kodi-cli-${version};`
`version = "1.1.1";`

We'll probably just do a big refactor of this anyway so there isn't a need to constrain this.

File: pkgs/applications/video/kodi-cli/default.nix
Comment (c0bw3b) [inline pkgs/applications/video/kodi-cli/default.nix]:
```suggestion
{ stdenv, fetchFromGitHub, makeWrapper, curl, bash, jq, youtube-dl, gnome3 }:
```

File: pkgs/applications/video/kodi-cli/default.nix
Comment (c0bw3b) [inline pkgs/applications/video/kodi-cli/default.nix]:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
```

File: pkgs/applications/video/kodi-cli/default.nix
Comment (c0bw3b) [inline pkgs/applications/video/kodi-cli/default.nix]:
```suggestion
    wrapProgram $out/bin/playlist_to_kodi --prefix PATH : ${stdenv.lib.makeBinPath [ curl bash gnome3.zenity jq youtube-dl ]}
```
(nitpick)

Comment (pstn) [reply pkgs/applications/video/kodi-cli/default.nix]: Rightly so :+1:

File: pkgs/applications/video/kodi-cli/default.nix
Comment (pstn) [reply pkgs/applications/video/kodi-cli/default.nix]: Oh, yeah. I always confuse those.

File: pkgs/applications/video/kodi-cli/default.nix
Comment (pstn) [reply pkgs/applications/video/kodi-cli/default.nix]: Seems like my indentation plugin is buggy. Need to fix that I guess.

**Comment by worldofpeace** [comment]
@pstn I've pushed basically reviewer nitpicks to your branch. I hope you don't mind.

Also, my apologies for your change being shelved on this `pname` discussion.
IMHO that shouldn't have been important as that change wasn't even on master yet.

**Comment by pstn** [comment]
@worldofpeace All good! I actually made the same point about waiting with a merge for this package myself. I also like the nitpicks, just had a busy week and no time to merge.

**Review by worldofpeace — APPROVED** [review]

**Pull request merged**
